server/src/compression.rs

```diff
@@ -459,6 +506,36 @@ mod tests {
             &test_util::compressed_schema(),
         )
         .unwrap();
+
+        assert_compressed_record_batch_with_segments_from_regular_time_series(
+            &uncompressed_timestamps,
+            &compressed_record_batch,
+            &[models::PMC_MEAN_ID],
+        )
+    }
+
+    #[test]
+    fn test_try_compress_irregular_almost_constant_time_series() {
+        let values = comp_test_util::generate_values(
+            TRY_COMPRESS_TEST_LENGTH,
+            comp_test_util::DataType::Constant,
```

Comment:
Should this be `comp_test_util::DataType::Random` to match `test_try_compress_regular_almost_constant_time_series()`? If so maybe also add a comment why `DataType::Random` is used to generate almost constant values.

Reply:
It should. Fixed.